exonum/src/runtime/types.rs

```diff
@@ -111,34 +117,42 @@ impl AnyTx {
 ///
 /// In string representation the artifact identifier is written as follows:
 ///
-/// `{runtime_id}:{artifact_name}`, where `runtime_id` is a [runtime identifier],
-/// and `artifact_name` is a unique name of the artifact.
+/// ```text
+/// {runtime_id}:{artifact_name}:{version}
+/// ```
+///
+/// where:
+///
+/// - `runtime_id` is a [runtime identifier],
+/// - `artifact_name` is the name of the artifact
+/// - `version` is the artifact semantic version
 ///
-/// Artifact name contains only the following characters: `a-zA-Z0-9` and one of `_-.:`.
+/// Artifact name may contain the following characters: `a-zA-Z0-9` and `_-.`.
```

Comment:
Fun fact: we're using regex-like ranges here, but in the second part (`_-.`) is follows the range pattern as well. To be a valid within regex, it should be `_.-`.